### PR TITLE
bugfix: monitoring causing OOM during kyma upgrade

### PR DESCRIPTION
**Pull request by AdrianPei**
**Description**
fix the bug that central monitoring causing OOM during kyma upgrade

Changes proposed in this pull request:

change the way to detect rma component installation status

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/internal/monitoring/monitoring.go [hunk 1]
```diff
@@ -102,25 +103,14 @@ func (c *client) IsPresent(releaseName string) (bool, error) {
...
+		return false, kebError.AsTemporaryError(err, "unable to get helm history")
 	}
-	for _, rel := range releases {
-		if rel.Name == releaseName {
-			return true, nil
-		}
-	}
-
-	return false, nil
+	return true, nil
```
Comment (ebensom) [inline components/kyma-environment-broker/internal/monitoring/monitoring.go:113]: You should also reimplement `IsDeployed` with a similar logic. After getting the helm release with the history action `.Run()`, you can check the current state of the release.

Comment (AdrianPei) [reply components/kyma-environment-broker/internal/monitoring/monitoring.go:115]: IsDeployed also fixed

**Review by ebensom — COMMENTED** [review]
/lgtm

You should also bump the KEB image in the chart

**Comment by AdrianPei** [comment]
> /lgtm
> 
> You should also bump the KEB image in the chart

KEB image bumped

**Pull request closed without merging**